roles/presclient/tasks/main.yml

```diff
@@ -0,0 +1,70 @@
+# windows updates
+#
+#
+#
+
+
+# set autologin
```

Comment:
Check the comments for the balloonprinter, so removing comments which are also in the `- name`, change desktop->Desktop